### ISSUE TITLE
latest version fails to build under OSX 10.10.1

### ISSUE THREAD

**Issue by xrokis** [mpv-player/homebrew-mpv#46]
I tried to update to the latest git version, but i'm not sure what the exact problem is.

```    
$ brew reinstall --HEAD mpv                                                                                                                                                    
==> Reinstalling mpv
==> Cloning https://github.com/mpv-player/mpv.git
Updating /Library/Caches/Homebrew/mpv--git
==> Checking out branch master
==> Patching
==> Downloading https://pypi.python.org/packages/source/d/docutils/docutils-0.11.tar.gz
Already downloaded: /Library/Caches/Homebrew/mpv--docutils-0.11.tar.gz
==> python setup.py install --prefix=/usr/local/Cellar/mpv/HEAD/libexec
==> Downloading http://ftp.waf.io/pub/release/waf-1.8.4
Already downloaded: /Library/Caches/Homebrew/mpv--waf-1.8.4
==> python waf configure --prefix=/usr/local/Cellar/mpv/HEAD --enable-zsh-comp
==> python waf install
Waf: Leaving directory `/private/tmp/mpv-N7GCnZ/build'
Build failed
 -> task in 'objects' failed (exit status 1):
	{task 4423720528: c gl_video.c -> gl_video.c.13.o}
['clang', '-D_ISOC99_SOURCE', '-D_GNU_SOURCE', '-D_LARGEFILE_SOURCE', '-D_FILE_OFFSET_BITS=64', '-D_LARGEFILE64_SOURCE', '-std=c99', '-Wall', '-O2', '-g', '-Werror=implicit-function-declaration', '-Wno-error=deprecated-declarations', '-Wno-error=unused-function', '-Wempty-body', '-Wdisabled-optimization', '-Wstrict-prototypes', '-Wno-format-zero-length', '-Werror=format-security', '-Wno-logical-op-parentheses', '-fcolor-diagnostics', '-Wno-tautological-compare', '-Wno-tautological-constant-out-of-range-compare', '-Wall', '-Wundef', '-Wmissing-prototypes', '-Wshadow', '-Wno-switch', '-Wno-parentheses', '-Wpointer-arith', '-Wredundant-decls', '-Wno-pointer-sign', '-pthread', '-I/private/tmp/mpv-N7GCnZ/build', '-I/private/tmp/mpv-N7GCnZ', '-I/usr/local/Cellar/little-cms2/2.6/include', '-I/usr/local/Cellar/ffmpeg/HEAD/include', '-I/usr/local/Cellar/mpg123/1.21.0/include', '-I/usr/local/include', '-I/usr/local/Cellar/freetype/2.5.5/include/freetype2', '-I/usr/local/Cellar/fribidi/0.19.6/include/fribidi', '-I/usr/local/Cellar/libass-ct/HEAD/include', '-DHAVE_LCMS2=1', '-DHAVE_LIBSWRESAMPLE=1', '-DHAVE_LIBAV=1', '-DHAVE_LIBAVFILTER=1', '-DHAVE_MPG123=1', '-DHAVE_LIBAVDEVICE=1', '-DHAVE_52=1', '-DHAVE_LIBASS=1', '../video/out/gl_video.c', '-c', '-o', '/private/tmp/mpv-N7GCnZ/build/video/out/gl_video.c.13.o']

READ THIS: http://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/homebrew):
  https://github.com/mpv-player/homebrew-mpv/issues
```

More detailed log: 
https://gist.github.com/e516db7e5f22beb94988

**Comment by twayz** [comment]
i also just got this error

**Comment by spaghetti2514** [comment]
Same here

**Comment by henry0312** [comment]
Thanks for your report.
There is nothing for me I can do because I haven't used OS X Yosemite‎ yet.
However, @pigoz may find something wrong.

**Comment by pigoz** [comment]
fixed with mpv-player/mpv@29cf62d
